config/locales/tooltips/en.yml

```diff
@@ -16,3 +16,7 @@ en:
       For your security, some government agencies require customer accounts to
       be verified using Personally Identifiable Information (PII) such as
       Social Security Number and proof of address.
+    authenticator_app: An authentication application is a mobile security application
```

Comment:
Should `authentication application` be `authenticator app` to match the main text on the screen?